camviewer_ui_impl.py

```diff
@@ -142,6 +142,8 @@ def __init__(self, app, main):
         self.last = QPoint(0, 0)
 
     def eventFilter(self, obj, event):
+        if event.type() in [QEvent.MouseButtonPress, QEvent.KeyPress]:
```

Comment:
I gave this a try, but the performance penalty was much, much worse than I expected. I'm not sure if this is intrinsic to qt/pyqt or a byproduct of how this app is architected, but if you turn on mouse tracking and look for `Enter` and/or `MouseMove` the image rate dips almost to zero whenever you exercise these events. I'm going to opt to not include these.

They did refresh the timer as expected fwiw

Reply:
Probably due to how frequently we receive a mouseMove event I suppose?  If each of those is allowed to trigger the callback, I can imagine that producing a lot of calls.  

I wonder if it'd help performance to also filter based on if we're in the "idle" state.  This is more of a musing, since that'd change the behavior of the app.